src/components/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -70,46 +65,32 @@
       </v-btn>
     </div>
 
-    <v-card v-if="showOrgPersonForm" flat class="people-roles-container">
-      <OrgPerson
-        v-show="showOrgPersonForm"
+    <div class="list-container px-4">
+      <!-- FUTURE: move OrgPerson inside ListPeopleAndRoles -->
```

Comment:
Right now you see the list or the add/edit component, but Scott would prefer to see add/edit inline with the table. Due to existing component architecture (ie, who does what), this is a large effort. I cleaned up code in preparation to do this but have not done it yet. After discussion with Scott I may create another ticket to do this change, which can get prioritized along with other work.

Reply:
/bcgov/entity#5076